refactor (list): Content update -vnext (#2335)
IgniteUI/igniteui-docfx: en/components/list.md

# Angular List View

<p class="highlight">

> The List element is extremely useful when presenting a group of items.Your list can be a simple list of textual items, or it can be a complex array of separate layout elements.

Reviewer: ```suggestion
The List element is extremely useful when presenting a group of items. You can create a simple list of textual items, or a more complex one, containing an array of different layout elements.
```

The Ignite UI for Angular List component displays rows of items and supports one or more header items as well as search and filtering of list items. Each list item is completely templatable and will support any valid HTML or Angular component. </p>
<div class="divider"></div>

> ## Angular grid list example

Reviewer: ```suggestion
## Angular grid list example
```
```suggestion
## Angular list example
```

# Angular List View

> <p class="highlight">

Reviewer 2: ```suggestion
```

Reviewer 2: I don't want to highlight only the first paragraph as it wont be consistent with the rest of the topics

The List element is extremely useful when presenting a group of items. You can create a simple list of textual items, or a more complex one, containing an array of different layout elements.

> The Ignite UI for Angular List component displays rows of items and supports one or more header items as well as search and filtering of list items. Each list item is completely templatable and will support any valid HTML or Angular component. </p>

Reviewer 2: ```suggestion
The Ignite UI for Angular List component displays rows of items and supports one or more header items as well as search and filtering of list items. Each list item is completely templatable and will support any valid HTML or Angular component.
```

> <div class="divider"></div>

Reviewer 2: ```suggestion
```

## Angular List Example

> 

Reviewer 2: ```suggestion
```

Reviewer 2: This is a removal of empty row

> The following example represents list populated with contacts with a name and a phone number, displayed under the name.The [`IgxList`]({environment:angularApiUrl}/classes/igxlistcomponent.html) component uses [**IgxAvatar**](avatar.md) and [**IgxIcon**](icon.md) to enrich the experience and add some functionality like picture avatar and icon for favorite a contact.In addition, the List View provides possibility of sorting achieved by using our filtering pipe.

Reviewer 2: ```suggestion
The following example represents a list populated with contacts with a _name_ and a _phone number_ properties. The [`IgxList`]({environment:angularApiUrl}/classes/igxlistcomponent.html) component uses [**IgxAvatar**](avatar.md) and [**IgxIcon**](icon.md) to enrich the user experience and expose the capabilities of setting avatar picture and different icon for _favorite a contact_. In addition, the List View expose sorting capabilities achieved by using our filtering pipe.
```

</div>
<div class="divider--half"></div>

> ## How to create list view win Angular?

Reviewer: ```suggestion
## How to create list view with Angular?
```

</igx-list-item>
```

> Looking for more filtering options? Check out our angular filtering options, including [**Excel Style Filtering**](grid/excel_style_filtering.md) and [**Advanced Filtering Options**](grid/advanced_filtering.md).

Reviewer: I don't think this paragraph is ok to be here, since the excel style filtering and the advanced filtering are in the context of the grid, not the list.
